src/pages/api/transactions.ts

```diff
@@ -80,4 +76,8 @@ export default async function handler(
         }
       }),
   );
+
```

Comment:
Shouldn't `transaction` be updated?

Reply:
Yes, I thought the front-end page was using SWR, so there would be another call soon, and therefore the returned data didn't need to be updated immediately.
But it seems that we can make a bit modification here to ensure that the returned transactions are updated after the update.